core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -177,6 +198,10 @@ var TopicsAndSkillsDashboardPage = function() {
       topicNameField,
       'Create Topic modal takes too long to appear.');
     await topicNameField.sendKeys(topicName);
+    await topicDescriptionField.sendKeys(description);
+    await topicCategoryField.click();
+    await (await
+    browser.driver.switchTo().activeElement()).sendKeys(category + '\n');
```

Comment:
Can we change the indentation here to make it more readable?

```suggestion
    await (
      await browser.driver.switchTo().activeElement()
    ).sendKeys(category + '\n');
```

Reply:
Done.